hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/RecordFinalizerBase.java

```diff
@@ -301,6 +309,33 @@ private static void updateNftChanges(
         final var currentNftChanges = nftChanges.get(nftId.tokenId());
         currentNftChanges.add(nftTransfer);
         nftChanges.put(nftId.tokenId(), currentNftChanges);
+
+        if (tokenRelChanges != null) {
+            final var receiverEntityIdPair = EntityIDPair.newBuilder()
+                    .accountId(receiverAccountId)
+                    .tokenId(nftId.tokenId())
+                    .build();
+            final var senderEntityIdPair = EntityIDPair.newBuilder()
+                    .accountId(senderAccountId)
+                    .tokenId(nftId.tokenId())
+                    .build();
+            if (isMint || isWipe) {
+                // The mint amount is not shown in the token transfer list. So for a mint transaction we need not show
+                // the token transfer changes to treasury
+                tokenRelChanges.remove(receiverEntityIdPair);
+                tokenRelChanges.remove(senderEntityIdPair);
+            } else {
+                // The mint amount is not shown in the token transfer list. So for a mint transaction we need not show
```

Comment:
Copy+paste comment I think? Maybe,
```
// Because the NFT transfer list already contains all information about how the
// sender/receiver # of owned NFT counts are changing, we don't repeat this in
// the tokenTransferLists; these merge() calls remove the duplicate information
```

Reply:
Yes, copy paste. Fixed. Thank you!